flp/README.md

```diff
@@ -0,0 +1,13 @@
+
+# flp
+A pool of ontologies for representing data about films and film contents
+
+##Contact:
+This space is administered by:
+
+**Diego Magro**
```

Comment:
Can you fix the render of the lines here?  Either make it a list or add trailing spaces to force line breaks.  Thanks.